app/client/views/sponsor/sponsor.html

```diff
@@ -337,6 +337,35 @@
     </div>
   </section>
 
+	<section id="sponsorship" style="display:flex; flex-direction:column; justify-content:space-around">
```

Comment:
inline css is generally considered bad practice, can we move these properties into `_sponsor.scss`?